### PR TITLE
xpro model additions

### PR DESCRIPTION
**Pull request by abeglova**
# What are the relevant tickets?
None

# Description (What does it do?)
This pr adds xpro columns that have been added since the data was staged

### CODE REVIEW

File: src/ol_dbt/models/staging/mitxpro/_stg_mitxpro__models.yml [hunk 1]
```diff
@@ -1104,6 +1112,9 @@ models:
     description: timestamp, specifying when a course run was initially created
   - name: courserun_updated_on
     description: timestamp, specifying when a course run was most recently updated
+  - name: courserun_certificate_available_date
```
Comment (rachellougee) [inline src/ol_dbt/models/staging/mitxpro/_stg_mitxpro__models.yml:1115]: Is this added to sql file?

Comment (abeglova) [reply src/ol_dbt/models/staging/mitxpro/_stg_mitxpro__models.yml:1115]: Oops, yeah it should be

Comment (abeglova) [reply src/ol_dbt/models/staging/mitxpro/_stg_mitxpro__models.yml:1115]: Actually nm I was working on both mitxonline and xpro changes  but decided to break it up into separate prs. certificate_available_date was actually added to mitxonlike i updated the wrong documentation file

**Review by rachellougee — APPROVED** [review]
👍

**Pull request merged**